data/governments.txt

```diff
@@ -193,6 +193,12 @@ government "Deep Security"
 government "Derelict"
 	"fine" 0
 
+government "Derelict (Boardable)"
+	"display name" "Derelict"
+	"player reputation" -1
+	"crew attack" 0
+	"crew defense" 0
```

Comment:
If you find a derelict ship, how will merchants treat it? Will they try to board/repair it?
Perhaps this government should be hostile to pirates so that pirates board them.

Reply:
if pirate board and loot them they'd be destroying them after so idk if thats what we want considering some are traps?

about the merchants I think we should make them hostile to these ships; they shouldnt board them and if they shoot a few its less worse?

Reply:
what if we use the marked personality for now?

Reply:
That might actually work.